core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -128,7 +128,10 @@ public class PGJobContextTest extends BasePGTest {
     private static final int count = 200;
     private static final String createDatesTblStmt = "create table xts as (select timestamp_sequence(0, 3600L * 1000 * 1000) ts from long_sequence(" + count + ")) timestamp(ts) partition by DAY";
     private static List<Object[]> datesArr;
+    private final Rnd bufferSizeRnd = TestUtils.generateRandom(LOG);
     private final boolean walEnabled;
+    private int minRecvBufferSize;
+    private int minSendBufferSize;
```

Comment:
This field seems to be always `0`. Why do we need it?

Reply:
ok, removed